src/drt/src/db/tech/frLayer.h

```diff
@@ -170,7 +166,7 @@ class frLayer
   frViaDef* getDefaultViaDef() const { return defaultViaDef; }
   bool hasVia2ViaMinStepViol() { return hasMinStepViol; }
   std::set<frViaDef*> getViaDefs() const { return viaDefs; }
-  dbTechLayerType getType() const { return type; }
+  dbTechLayerType getType() const { return (fakeCut)? dbTechLayerType("cut") : (fakeMs)? dbTechLayerType("masterslice") : db_layer_->getType(); }
```

Comment:
Use the enum values rather than string lookup, e.g.

dbTechLayerType("masterslice") -> dbTechLayerType::MASTERSLICE